view/main/index.css

```diff
@@ -80,21 +80,17 @@ body {
   color: inherit;
   text-decoration: none;
 
-  &:hover,
-  &.is-hover {
-    color: var(--accent);
-    text-decoration: underline;
-  }
-
   &:focus {
     text-decoration: none;
     border-radius: 2px;
     outline-offset: 2px;
   }
 }
 
-.main_logo-path {
-  fill: var(--surface-ui-regular);
+.main:has(.main_logo:hover) .main_link--em,
```

Comment:
No. I mean replace `.is-em` to `.main_link.is-em`

Reply:
Okay. Please could you use `suggestions` (if you review via laptop)?
Because there are classic BEM modifiers like `main_row--sample` and class combination like `is-em`

Reply:
I use it from laptop, but now I am from phone in the airport. I didn't have screen to find the element name 😅